src/main/java/uk/gov/cshr/controller/OrganisationController.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.cshr.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.oauth2.provider.OAuth2Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import uk.gov.cshr.domain.Identity;
+import uk.gov.cshr.service.organisation.OrganisationDto;
+import uk.gov.cshr.service.organisation.OrganisationService;
+import uk.gov.cshr.service.security.IdentityService;
+
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+@RequestMapping("/organisation")
+@PreAuthorize("hasPermission(returnObject, 'read')")
+public class OrganisationController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrganisationController.class);
+
+    @Autowired
+    private OrganisationService organisationService;
+
+    @Autowired
+    private IdentityService identityService;
+
+    @GetMapping
+    public String getOrganisation(Model model) {
+
+        List<OrganisationDto> organisations = organisationService.getOrganisations();
+        model.addAttribute("organisations", organisations);
+
+        return "organisation/add";
```

Comment:
Method name "getOrganisation" suggests that it is to fetch the organisations but the return value "organisation/add" is adding the organisation. Could you please correct the return value.

Reply:
It get all the organisations from DB and then redirect to that add page. add here is html page and return redirects to add.html. Also its get mapping as it gets organisations.
So to me it looks good.